src/clevertap.js

```diff
@@ -26,6 +26,9 @@ export default class CleverTap {
     }
     Account.setAccountId(id);
     Account.setRegion(region);
+    if(Device.getVAPIDState === null){
```

Comment:
`getVAPIDState` is a function. Need to update above line to `if(Device.getVAPIDState() === null)`